request_url.go

```diff
@@ -43,7 +43,11 @@ func createRequestUrl(method, host, path, key, secret, timestamp string, secure
 	params.Add("auth_signature", authSignature)
 
 	if host == "" {
-		host = "api.pusherapp.com"
+    if cluster != "" {
```

Comment:
You can use go fmt to format this. The Go standard is to use 8 tabs for indentation. You can just run `go fmt` on the project.